python/src/dataset.rs

```diff
@@ -53,6 +54,91 @@ const DEFAULT_NPROBS: usize = 1;
 const DEFAULT_INDEX_CACHE_SIZE: usize = 256;
 const DEFAULT_METADATA_CACHE_SIZE: usize = 256;
 
+#[pyclass(name = "_Operation", module = "_lib")]
+#[derive(Clone)]
+pub struct Operation {
+    inner: LanceOperation,
+}
```

Comment:
nit: if this is just a lightweight wrapper, can just write as a tuple struct:

```suggestion
pub struct Operation(LanceOperation)
```

Reply:
Good idea.  Done.